packages/@okta/vuepress-site/docs/concepts/interaction-code/index.md

```diff
@@ -0,0 +1,100 @@
+
+The Interaction Code grant is intended for developers who want a step-by-step remediation user experience without redirecting to an authorization server. This grant type enables developers to include [Identity Engine features](https://help.okta.com/okta_help.htm?type=oie&id=ext-features) in their app, such as passwordless authentication and progressive profiling. The Interaction Code flow can use policy-driven remediation steps not only for authentication, but also for registration and account recovery. See [Redirect authentication vs. embedded authentication](/docs/concepts/redirect-vs-embedded/) for Identity Engine authentication deployment models and [Identity Engine deployment guides](/docs/guides/oie-intro/) for detailed deployment use cases.
+
+## The Interaction Code flow
+
+The Interaction Code flow is similar to the [OAuth 2.0 Authorization Code flow with PKCE](/docs/concepts/oauth-openid/#authorization-code-flow-with-pkce). All clients are required to pass along a client ID, as well as a Proof Key for Code Exchange (PKCE), to keep the flow secure. Confidential clients such as web apps must also pass a client secret in their authorization request. The user can start the authorization request with minimal information, relying on the client to facilitate the interactions with the Okta org to progressively authenticate the user. The series of interactions, which could include multifactor authentication steps, is secured using the `interaction_handle`. After successfully completing the remedial interactions, the client receives an `interaction_code` that they can then redeem for tokens from an Okta Authorization Server with the [`/token`](/docs/reference/api/oidc/#token) endpoint.
+
+The Interaction Code flow is supported by Okta's two authorization server types: the [Org Authorization Server](/docs/concepts/auth-servers/#org-authorization-server) and [Custom Authorization Server](/docs/concepts/auth-servers/#custom-authorization-server). These Okta Authorization Servers must have the Interaction Code grant type configured in their Access Policy Rule to accept Interaction Code requests.
```

Comment:
It might be nice to link "Access Policy Rule" to somewhere like https://developer.okta.com/docs/guides/customize-authz-server/create-access-policies/ for more information.